src/main/java/seedu/address/logic/parser/AddSellerCommandParser.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CHARACTERISTICS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PRICE_RANGE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROPERTIES;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.AddSellerCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.role.Seller;
+
+/**
+ * Parses input arguments and creates a new AddBuyerCommand object
```

Comment:
Found some discrepancies in the JavaDocs
```suggestion
 * Parses input arguments and creates a new AddSellerCommand object
```